apis/crd/v1alpha1/cloudprovideraccount_webhook.go

```diff
@@ -74,40 +74,35 @@ func (r *CloudProviderAccount) ValidateCreate() error {
 		}
 
 		// warning for using role based auth
-		if len(strings.TrimSpace(awsConfig.RoleArn)) != 0 {
-			cloudprovideraccountlog.Info("Role ARN configured will be used for cloud-account access")
-			// empty credentials when role based access is configured
-			awsConfig.AccessKeyID = ""
-			awsConfig.AccessKeySecret = ""
-		} else if len(strings.TrimSpace(awsConfig.AccessKeyID)) == 0 || len(strings.TrimSpace(awsConfig.AccessKeySecret)) == 0 {
-			return fmt.Errorf("must specify either credentials or role arn, cannot both be empty")
-		}
+		//if len(strings.TrimSpace(awsConfig.RoleArn)) != 0 {
```

Comment:
@bangqipropel take a look at these changes, and see how you can enable webhook for it